### DISCUSSION TITLE
Bug in Tray plugin : track name not showing in monitor

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#878] [Help]
Happens on Studio one, the track name for AG inserts  are missing in the connection tray popup. 
Plugins path and statistics are shown.

Found the problem on Studio One 5.4 for now. Issue doesnt exists with cubase.
So probably an issue with how to get track names in Studio one

**Comment by apohl79** [comment]
Are you using VST3 in S1? VST2 does not support this.

**Comment by Henri75** [comment]
Yes VST3 !

**Reply by apohl79** [reply]
Maybe S1 does not use the VST3 host interface for this, I don't know. AG just reads from that property.
